File: C++/Programming Basics with C++/07. Exercise - Nested Conditional Statement/simple_task/SummerOutfit.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {

    int degrees;
    string dayTime;
    cin >> degrees >> dayTime;

    string outfit;
    string shoes;
    if (degrees >= 10 && degrees <= 18) {
        if (dayTime == "Morning") {
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (dayTime == "Afternoon" || dayTime == "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    } else if (degrees > 18 && degrees <= 24) {
        if (dayTime == "Morning" || dayTime == "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (dayTime == "Afternoon") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
    } else if (degrees >= 25) {
        if (dayTime == "Morning") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        } else if (dayTime == "Afternoon") {
            outfit = "Swim Suit";
            shoes = "Barefoot";
        } else if (dayTime == "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    }

    cout << "It's " << degrees << " degrees, get your " << outfit << " and " << shoes <<"."<< endl;
    return 0;
}

File: C#/Programming Basics with C#/07. Nested Conditional Statements - Exercise/04. Summer Outfit/SummerOutfit.cs
using System;

class SummerOutfit
{
    static void Main()
    {
        int degrees = int.Parse(Console.ReadLine());
        string dayTime = Console.ReadLine().ToLower();

        if (degrees >= 10 && degrees <= 18)
        {
            if (dayTime == "morning")
            {
                Console.WriteLine($"It's {degrees} degrees, get your Sweatshirt and Sneakers.");
            }
            else if (dayTime == "afternoon" || dayTime == "evening")
            {
                Console.WriteLine($"It's {degrees} degrees, get your Shirt and Moccasins.");
            }
        }
        if (degrees > 18 && degrees <= 24)
        {
            if (dayTime == "morning" || dayTime == "evening")
            {
                Console.WriteLine($"It's {degrees} degrees, get your Shirt and Moccasins.");
            }
            else if (dayTime == "afternoon")
            {
                Console.WriteLine($"It's {degrees} degrees, get your T-Shirt and Sandals.");
            }
        }
        if (degrees >= 25)
        {
            if (dayTime == "morning")
            {
                Console.WriteLine($"It's {degrees} degrees, get your T-Shirt and Sandals.");
            }
            else if (dayTime == "afternoon")
            {
                Console.WriteLine($"It's {degrees} degrees, get your Swim Suit and Barefoot.");
            }
            else if (dayTime == "evening")
            {
                Console.WriteLine($"It's {degrees} degrees, get your Shirt and Moccasins.");
            }
        }
    }

}